test/acceptance/objects/setup_test.go

```diff
@@ -20,74 +20,72 @@ import (
 )
 
 func Test_Objects(t *testing.T) {
-	t.Run("setup", func(t *testing.T) {
-		createObjectClass(t, &models.Class{
-			Class: "TestObject",
-			ModuleConfig: map[string]interface{}{
-				"text2vec-contextionary": map[string]interface{}{
-					"vectorizeClassName": true,
-				},
+	// t.Run("setup", func(t *testing.T) {
```

Comment:
did you intend to leave this commented?

Reply:
I think that this line should be uncommented.

Reply:
@antas-marcin can you please explain why this line should be?

@parkerduckworth I didn't remove this line in order to raise the question of why this line is commented.

This pattern of putting the test setup into t.run() repeats itself though the code.

In our case all sub-tests depend on this setup in order to work correctly. In other words if the setup fails then all sub-tests will fail. This is why the setup needs to be called in the main goroutine because this is a prerequisite in order for all sub-tests to run properly.

Since this is more like a pattern I think it's worth to discuss this in our next meeting.

Reply:
I think that the intent of this `t.Run("setup",)` was just to logically separate this part of the tests.

I think that tests behave the same, no matter if it's called in the main goroutine or inside `t.Run("setup", )`, but yeah, let's talk it over.